core/nodejsAction/Dockerfile

```diff
@@ -2,56 +2,20 @@ FROM buildpack-deps:trusty-curl
 
 ENV DEBIAN_FRONTEND noninteractive
 
-# Initial update and some basics.
-# This odd double update seems necessary to get curl to download without 404 errors.
-RUN curl -sL https://deb.nodesource.com/setup_0.12 | bash - && \
-  apt-get install -y nodejs imagemagick
-
```

Comment:
the packages below are installed as part of the Docker image rather than package.json to separate the packages needed for running the proxy service vs those we include in the image for the user actions.